src/Sia.Gateway/Initialization/ServicesStartup.cs

```diff
@@ -100,6 +100,7 @@ public static void AddThirdPartyServices(this IServiceCollection services, IConf
 
             //Adds every request type in the Sia.Gateway assembly
             services.AddMediatR(typeof(GetIncidentRequest).GetTypeInfo().Assembly);
+            services.AddPipelineBehavior();
```

Comment:
It might be best to name this AddMediatRPipelineBehavior, just to make it more clear that this is related to MediatR.